xtable-core/src/test/resources/junit-platform.properties

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-junit.jupiter.execution.parallel.enabled=true
+junit.jupiter.execution.parallel.enabled=false
```

Comment:
I have run the tests locally by setting this to true and they pass, can we revert this config and see the GH build?